ynr/apps/candidates/models/db.py

```diff
@@ -122,6 +127,10 @@ def diff_html(self):
         except VersionNotFound as e:
             return "<p>{}</p>".format(escape(text_type(e)))
 
+    def save(self, **kwargs):
+        set_review_required(self)
```

Comment:
Any reason why we're passing an instance of `LoggedAction` out to another function (which can only sensibly take an instance of `LoggedAction`) to set a property on it rather than just defining `set_review_required()` as a class method on `LoggedAction`?

i.e: `set_review_required(self)` vs `self.set_review_required()`